src/core/border_router/routing_manager.cpp

```diff
@@ -3700,6 +3734,7 @@ void RoutingManager::PdPrefixManager::EvaluateStateChange(Dhcp6PdState aOldState
     {
     case kDhcp6PdStateDisabled:
     case kDhcp6PdStateStopped:
+    case kDhcp6PdStateIdle:
```

Comment:
Need adhoc changes for PdStateChange D-Bus API

Reply:
Done